package.json

```
@@ -14,7 +14,10 @@
     "react-dom": "17.0.2"
   },
   "devDependencies": {
+    "autoprefixer": "^10.4.4",
     "eslint": "8.11.0",
-    "eslint-config-next": "12.1.0"
+    "eslint-config-next": "12.1.0",
+    "postcss": "^8.4.12",
+    "tailwindcss": "^3.0.23"
```

```suggestion
    "postcss": "8.4.12",
    "tailwindcss": "3.0.23"
```
(same suggestion for "autoprefixer")

In order to assure better control of updates, we usually preffer using specific versions (and running bots to create PR with the updates). This way, all contributors of a project are ensured to work under the exact same versions AND we still have a frequent "minor fixes" automation from the bot (not needed for the Internal Certification project).

---

@malparty , I like the idea of specific versioning
however, regarding
> all contributors of a project are ensured to work under the exact same versions
>
isn't `yarn.lock` ensuring it already?

---

Sorry, you are absolutely true, my example was not the good one! 🙌 

Usually, we don't review the lock file (god thanks) so removing the caret ensures we see any version change. There are still, even in minor versions, some rare breaking or unwanted changes.

Again, for an IC it is not that important, but when the project gets bigger, it can save hours of debugging :)

---

yeah, I now totally understand, usually `yarn.lock` lurks in a corner in code review and is missed often due to the number of line changes 😄  updated to specific version 👍